// path-parse/test.js
//  front\pages\index.js
import React, { useState, useRef, useEffect, useState } from 'react';
import { Button, Form, Input, PostCard, Icon, Card, Avatar } from 'antd';


const dummy = {
	isLoggedIn: true,
	imagePaths: [],
	mainPosts: [{
		User: {
			id: 1,
			nickname: '제로초',
		},
		content: '첫 번째 게시글',
		img: 'https://bookthumb-phinf.pstatic.net/cover/137/995/13799585.jpg?udate=20180726',
	}],
};

const Home = () => {
	return (
		<>
			<div>
				<Form style={{ margin: '10px 0 20px' }} encType="multipart/form-data">
					<Input.TextArea maxLength={140} placeholder="어떤 신기한 일이 있었나요?" />
					<div>
						<input type="file" multiple hidden />
						<Button>이미지 업로드</Button>
						<Button type="primary" style={{ float: 'right' }} htmlType="submit">짹짹</Button>
					</div>
				</Form>,

		        {dummy.mainPosts.map((c) => {
					return (
						<Card
							key={+c.createdAt}
							cover={c.img && <img alt="example" src={c.img} />}
							actions={[
								<Icon type="retweet" key="retweet" />,
								<Icon type="heart" key="heart" />,
								<Icon type="message" key="message" />,
								<Icon type="ellipsis" key="ellipsis" />,
							]}
							extra={<Button>팔로우</Button>}
						>

							<Card.Meta
								avatar={<Avatar>{c.User.nickname[0]}</Avatar>}
								title={c.User.nickname}
								description={c.content}
							/>
						</Card>
					);
				})}
			</div>
		</>
	);
};

export default Home;
